modules/exploits/linux/http/cisco_rv340_lan.rb

```diff
@@ -0,0 +1,190 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cisco RV34X Series Authentication Bypass and Command Injection',
+        'Description' => %q{
+          This module exploits two vulnerabilities, a session ID directory traversal authentication
+          bypass (CVE-2022-20705) and a command injection vulnerability (CVE-2022-20707), on Cisco RV340 and RV345
+          Small Business Routers, allowing attackers to gain remote shell access with www-data user privileges.
```

Comment:
```suggestion
          Small Business Routers, allowing attackers to execute arbitrary commands with www-data user privileges.
```

Reply:
This is important since in theory they could use it to gain a shell, but they could also use it for other activities that don't require a shell.